### PR TITLE
Added a Google Doc Light Theme

### PR DESCRIPTION
**Pull request by qennenat**
Hello!

First and foremost, great job and I sincerely appreciate the extension that you made. It has been tremendously helpful so far.

I extended your original implementation to include a "Google Doc Light" theme in which there is **no** syntax highlighting (a.k.a. all syntax highlighting is black) to reflect the nature of a Google interview where you are expected to [write code on a Google Doc](https://www.teamblind.com/post/Google-interview--google-docs-DfH7GSfP).

Preview...
image

Thank you!

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -41,6 +41,7 @@ <h4 id="selectedTheme"></h4>
         <option value="cm-s-eclipse">Eclipse</option>
         <option value="cm-s-elegant">Elegant</option>
         <option value="cm-s-erlang-dark">Erlang Dark</option>
+        <option value="cm-s-google-doc-light">Google Doc Light</option>
```
Comment (vishnuvardhan-s) [inline popup.html]: I really like this idea. Can we move this as the **first** theme, I know I had everything alphabetical, but this seems to be useful for folks preparing for Google.  Also if possible, can we name it like "Google Interview" or something catchy 😄

Comment (qennenat) [reply popup.html]: Sure! I'm glad you like it. 😁 

I am little busy handling some errands but I will get it done as soon as possible. I am also planning on creating a "Google Interview - Night" style also. 

**Action Items**:
- [x] Move "Google Interview" __and__ "Google Interview - Night" to top of dropdown list
- [x] Add a "Google Interview - Night" style

Comment (vishnuvardhan-s) [reply popup.html]: Great! Take your time!

Comment (qennenat) [reply popup.html]: Hey Vishnu, hope all is well.

So, I just started again going through and implementing the changes. However, I noticed in my initial implementation of **_Google Interview Day_**, the CSS style wrapper via Code-Mirror is **not** being generated.

Would you happen to know why? Its been puzzling me for a while now.

For example, if I apply the theme **_Abbott_**, the style is applied correctly.

### Example of _Abbot_ ✅ 
image

### Example of _Google Interview Day_ ❌ 
image

**Comment by qennenat** [comment]
As requested, I completed it by adding **Google Interview Day** and **Google Interview Night**...

Screenshot (2)

**Review by vishnuvardhan-s — APPROVED** [review]
LGTM!

**Comment by vishnuvardhan-s** [comment]
Thanks for the work that you did. Wil let you know once this change is available in Chrome Web Store!

File: popup.html (hunk 1 above)
Comment (vishnuvardhan-s) [reply popup.html]: I guess you got the issue, there were some hidden 3024 classnames 😁

Comment (qennenat) [reply popup.html]: Yep, my bad. 🤦🏼 

Was going crazy until I realized it!

**Comment by vishnuvardhan-s** [comment]
Hi @edmund-leibert, I submitted the changes and now your changes are available in Chrome Store 🎉

**Pull request merged**